app/services/draft_case_contact_cleanup_service.rb

```diff
@@ -0,0 +1,12 @@
+class DraftCaseContactCleanupService
+  def self.call
+    draft_expiration = 1.week.ago
+    draft_expiration_without_case = 1.day.ago
```

Comment:
I meant constants like `DRAFT_EXPIRATION_DELAY` and `ORPHANED_DRAFT_EXPIRATION_DELAY`

Not sure if orphaned is a better name